From 6f5f0496519fd9ccc49bbc8182e42f3a35ca23b7 Mon Sep 17 00:00:00 2001
From: Brett Jia <dev.bjia56@gmail.com>
Date: Wed, 28 Feb 2024 16:25:12 -0500
Subject: [PATCH] Update gtest_main.cc

---
 crypto/test/gtest_main.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/test/gtest_main.cc b/crypto/test/gtest_main.cc
index 26ed321c0a..a0535f6a81 100644
--- a/crypto/test/gtest_main.cc
+++ b/crypto/test/gtest_main.cc
@@ -43,7 +43,7 @@ int main(int argc, char **argv) {
     }
 #endif
 
-#if defined(TEST_ARM_CPUS)
+#if defined(TEST_ARM_CPUS) && !defined(OPENSSL_STATIC_ARMCAP)
     if (strncmp(argv[i], "--cpu=", 6) == 0) {
       const char *cpu = argv[i] + 6;
       uint32_t armcap;